samples/suit/smp_transfer/Kconfig

```diff
@@ -4,8 +4,8 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-config SUIT_ENVELOPE_SEQUENCE_NUM
-	int "Sequence number of the SUIT envelopes"
+config N_BLINKS
+	int "Number of fast blinks"
```

Comment:
It seems it is possible to access "$(VERSION_MAJOR)" from a Kconfig file. Couldn't we use the same mechanism also for `app_root_seq_num` to set the default value of `N_BLINKS`? I am afraid that we will waste time inside the team if we forget to update this variable ;)

Reply:
Discussed offline: It is hard to decide if major, minor or patch should be exposed. On the other hand, `APP_ROOT_SEQ_NUM` is unavailable in Kconfigs and the app and exposing it through CMake magic will make a lot of mess just to sync them.